dags/ped_data_pub.py

```diff
@@ -0,0 +1,80 @@
+# test locally with: docker compose run --rm airflow-cli dags test atd_trail_counters
+
+import os
+from datetime import datetime, timedelta
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+from utils.onepassword import load_dict
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+default_args = {
+    "owner": "airflow",
+    "description": "Scrapes trail counter data from the public eco-counter website and publishes it in Socrata",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+docker_image = "atddocker/atd-trail-counters:latest"
+
+REQUIRED_SECRETS = {
+    "SO_WEB": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.endpoint",
+    },
+    "SO_PASS": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeySecret",
+    },
+    "SO_USER": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeyId",
+    },
+    "COUNTERS_DATASET": {
+        "opitem": "Socrata Dataset IDs",
+        "opfield": "datasets.Trail Counters",
+    },
+}
+
+with DAG(
+    dag_id="atd_trail_counters",
```

Comment:
it'd be nice to have this DAG's file name be more similar to the DAG name. e.g. `atd_trail_counters.py`